### ISSUE TITLE
Why is dagu reading irrelevant YAML files?

### ISSUE THREAD

**Issue by kseistrup** [yohamta/dagu#199]
I started `dagu` with `dagu server` in my homedir [on Linux], and now the webui asks me to check three “errors below”. Those “errors” are — according to `dagu` — incompatible YAML errors, but why on Earth is `dagu` reading YAML files that has got nothing to do with `dagu`? Is it simply scanning the homedir and reading each and every YAML file it can find?

**Comment by yohamta** [comment]
Dagu read YAML files placed in the current directory unless you set the `dags` config in the admin config file.
Please create `~/.dagu/admin.yaml`.

```yaml
host: <hostname for web UI address>                          # default value is 127.0.0.1
port: <port number for web UI address>                       # default value is 8000
dags: <the location of DAG configuration files>              # default value is current working directory
```

I suppose the default DAG directory should not be current directory but other directory so yaml files are not mixed up, for example `~/.dagu/dags`. We'll fix this soon.

**Comment by kseistrup** [comment]
Thanks.

`dagu` should definitely not find random YAML files in the homedir.

Using `~/.dagu/dags` is acceptable, but you should consider following the XDG Base Directory Specification:

* https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
* https://wiki.archlinux.org/title/XDG_Base_Directory

That way configuration files would per default go to `~/.config/dagu/` and DAGs could go in `~/.local/share/dagu/` (unless, of course, the `$XDG_*` variables says otherwise.

**Comment by yohamta** [comment]
@kseistrup Thanks a lot! We will modify the default paths to conform to the standard (XDGBDS) while also checking the specifications of other OSS such as Airflow.

> PS: You may want to mention the “Dagu reads YAML files placed in the current directory” in the quick start so that people are not in for a surprise if the follow the quick start without having read the entire README.

I updated the quick start in the README.

**Comment by yohamta** [comment]
This issue is fixed by #207 
We use `~/.dagu/dags` as the default path for DAGs for now to maintain consistency.
But we may change it in the future major version release.
